setup.py

```diff
@@ -15,6 +15,8 @@
     from distutils.core import Extension
     from distutils.core import setup
 from distutils.command.build_ext import build_ext
+from distutils.dep_util import newer_group
```

Comment:
Let's try to import packages/modules instead of functions:
https://google.github.io/styleguide/pyguide.html#Imports